concordances/test_fixtures.go

```diff
@@ -0,0 +1,112 @@
+package concordances
+
+//This file contains common fixtures shared by all test files in this package
+import (
+	db "github.com/Financial-Times/concordances-rw-dynamodb/dynamodb"
+
+	"errors"
+	"github.com/Financial-Times/concordances-rw-dynamodb/sns"
+)
+
+const (
+	DDB_ERROR     = "DynamoDB error"
+	SNS_ERROR     = "SNS error"
+	EXPECTED_UUID = "uuid_123"
+)
+
+var oldModel = db.ConcordancesModel{UUID: EXPECTED_UUID, ConcordedIds: []string{"A", "B"}}
+var updateModel = db.ConcordancesModel{UUID: EXPECTED_UUID, ConcordedIds: []string{"A"}}
+
+type MockSNSClient struct {
+	Happy   bool
+	Invoked bool
+}
+
+func (c *MockSNSClient) SendMessage(uuid string) error {
+	c.Invoked = true
+	if c.Happy {
+		return nil
+	}
+	return errors.New(SNS_ERROR)
+}
+
+func (c *MockSNSClient) Healthcheck() (bool, error) {
+	c.Invoked = true
+	if c.Happy {
+		return true, nil
+	}
+	return false, errors.New(SNS_ERROR)
+}
+
+type MockDynamoDBClient struct {
+	Happy bool
+	model db.ConcordancesModel
+}
+
+func (ddb *MockDynamoDBClient) Read(uuid string) (db.ConcordancesModel, error) {
+	if ddb.Happy {
+		return oldModel, nil
+	}
+	return db.ConcordancesModel{}, errors.New(DDB_ERROR)
+}
+
+func (ddb *MockDynamoDBClient) Write(m db.ConcordancesModel) (db.ConcordancesModel, error) {
+	if !ddb.Happy {
+		return db.ConcordancesModel{}, errors.New(DDB_ERROR)
+	}
+
+	if ddb.model.UUID == "" {
+		ddb.model = m
+		return db.ConcordancesModel{}, nil
+
+	}
+	ddb.model = m
+	return oldModel, nil
+}
+
+func (ddb *MockDynamoDBClient) Delete(uuid string) (db.ConcordancesModel, error) {
+	if !ddb.Happy {
+		return db.ConcordancesModel{}, errors.New(DDB_ERROR)
+	}
+	if ddb.model.UUID == "" {
+		return db.ConcordancesModel{}, nil
+	}
+	return oldModel, nil
+}
+func (ddb *MockDynamoDBClient) Count() (int64, error) {
```

Comment:
Count isn't being implemented, so it should be removed from the client interface/implementations.